### ISSUE TITLE
WEBGUI cert script error

### ISSUE THREAD

**Issue by Nano0k** [kvic-z/pixelserv-tls#43]
After many successful uses of your great script, the last couple have resulted in an error of:

-sh: $: not found

I'm still able to establish a secure connection to router.asus.com, however, but still wonder why tho?

Thanks!

**Comment by kvic-z** [comment]
Good to hear there are still users of my config-webgui.sh out there!

There was a configuration error on my end in a recent (perhaps as early as the turn of the year) server infra re-structuring. This error is fixed now. So you should be able to run the script again. Thanks for letting me know the issue.

The reason router.asus.com continues to work because the server certificate generated from config-webgui.sh is valid for ten years (if I recall correctly, you can check the certificate in your browser by clicking the padlock icon). You only have to re-run the script if there is change on your end. Otherwise, the server certificate generated and configured by the script will continue to work.

Cheers

**Comment by Nano0k** [comment]
Weird, entering

$ sh -c "$(wget -qO - https://kazoo.ga/pixelserv-tls/config-webgui.sh)"

returns the same error. Syntax error? Perhaps anything to do with the router-based VPN (never did before)? Or anything else on my end, like a reboot?

Thanks again!

**Comment by kvic-z** [comment]
Unfortunately I can't reproduce the issue. I tested it from a computer in the cloud so it is just like from any Asus router trying to access the script. The test succeeded with showing me the screen prompt to proceed further (which I won't do of cos as it's not an Asus router).

Possibly a DNS issue on your end? Such as "kazoo.ga" being blocked by your service provider. Just some food for thought. You can quickly check if you're able to load "https://kazoo.ga" in your browser.

**Comment by Nano0k** [comment]
All I have to say to that is "Kazoo It!" so that worked. Oh well, I have actual years to figure this out. :)
